client/blocks/login/social.scss

```diff
@@ -67,7 +67,7 @@
 							max-width: 240px;
 							text-align: center;
 							margin-right: auto;
-							margin-left: -20px;
+							margin-left: -10px;
```

Comment:
Removing this entirely looks better on desktop but it looks off center with the separator on mobile.